webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/zarr3/Zarr3ArrayHeader.scala

```diff
@@ -251,13 +252,20 @@ object Zarr3ArrayHeader extends JsonImplicits {
       )
 
   }
-  def fromDataLayer(dataLayer: DataLayer): Zarr3ArrayHeader = {
+  def fromDataLayer(dataLayer: DataLayer, mag: Vec3Int): Zarr3ArrayHeader = {
     val additionalAxes = reorderAdditionalAxes(dataLayer.additionalAxes.getOrElse(Seq.empty))
+    // TODOM: Make code together with ZarrHeader.scala DRYer
```

Comment:
I copied over this code from `ZarrHeader.scala`. @fm3 Do you think it makes sense to invest some time to extracted shared code from both files to make the code DRYer or do you think it is ok as it is now (and I just remove this comment here)?

Reply:
I think it’s fine

Reply:
Ok I just removed the comment :)